### DISCUSSION TITLE
ASReview Decision Tree for Students

### DISCUSSION THREAD

**Discussion by MaxvanHaastrecht** [asreview/asreview#1370] [Show and tell]
Hey everyone👋 I've noticed that students that are just getting started with ASReview, and systematic reviewing in general, often make mistakes that feel like they could have been prevented. It costs them a lot of time to correct these mistakes. I think it's a shame when someone's first experience with systematic reviewing is a frustrating one.

That led me to the idea of creating a brief series of questions for students to see if they are ready to get started with ASReview. A sort of 'ASReview Decision Tree' if you will. I have attached my first draft here and have also made it available on Google Drive where you should be able to add comments to the [PDF document](https://drive.google.com/file/d/1G1CHgVyJPX_WypyrlZ0QPYcP8iP8zeYJ/view?usp=sharing).

Any feedback on this first version is very much appreciated. Am I missing out on some crucial resources? Should certain questions be replaced with other ones? I will try to keep it somewhat up-to-date based on your suggestions. In the meantime, feel free to use this version or create your own version. Hopefully we can help everyone to have a positive first experience with ASReview!

[ASReview Decision Tree.pdf](https://github.com/asreview/asreview/files/11052313/ASReview.Decision.Tree.pdf)

![ASReview Decision Tree](https://user-images.githubusercontent.com/65038675/227247971-51b8ec30-5a4f-4b07-bdc0-5e338d71a01c.png)

**Comment by J535D165** [comment]
Very nice!!

**Comment by rohitgarud** [comment]
Great... This will be very helpful in reducing the barriers to entry in using ASReview for not just extensive Systematic Reviews but even for literature reviews for the thesis.
